LogcatCoreLib/src/main/res/values/styles.xml

```diff
@@ -1,6 +1,6 @@
 <resources>
 
-    <style name="DetailTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="DetailTheme" parent="Theme.MaterialComponents.Light.DarkActionBar.Bridge">
```

Comment:
Please can you tell me the benefit. I don't know that

Reply:
It's the new material theme, for example new Android Studio projects will use `Theme.MaterialComponents`. The benefit is theme alignment, subtle differences in appearance if any... I don't think there are any benefits for this project really, you can revert to using `Theme.AppCompat`. I use a custom theme myself, I see the theme in the lib more as an example, and a custom theme should be defined by the lib user based on this example.

Reply:
Ahh sorry, I've to specify my question. What's the `.Bridge` ?

Reply:
`Bridge themes inherit from AppCompat themes, but also define the new Material Components theme attributes for you. If you use a bridge theme, you can start using Material Design components without changing your app theme.`

Basically, Bridge themes set defaults for attributes used in MaterialComponents styles, so you don't have to define most or all of them in the theme, I believe that is the main benefit.